leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -551,8 +581,9 @@ public void showLoadLocationError() {
   }
 
   @Override
-  public void willGetLocationInfo() {
-    changeLocationInfoLayoutVisibility(View.GONE);
+  public void willGetLocationInfo(double lng, double lat) {
```

Comment:
as ISO 6709 says, latitude comes before longitude. Please, change in all the PR the order to respect the ISO.
https://en.wikipedia.org/wiki/ISO_6709
And try to use the full word, latitude and longitude instead lat lng